dna/__main__.py

```diff
@@ -575,6 +579,9 @@ def get_train_and_test_data(
 
 
 def get_ootsp_split_data(train_data, test_data, split_ratio, split_seed):
+    # TODO: Why is `split_ratio` necessary here? It only seems to be
+    # reducing the size of the training data set. Also, when `split_ratio < 1`,
+    # we can no longer ensure that all the primitives are present in the training set.
```

Comment:
@bjschoenfeld my friend do you know if the `split_ratio` argument is necessary here? From what I can tell, all it does is reduce the size of the training data set. Having it here, we can't ensure all primitive are present in the training set.

Reply:
I am not too worried about all the ootsp stuff right now.